torch/utils/_sympy/value_ranges.py

```diff
@@ -69,6 +69,11 @@ def __init__(self, lower, upper):
 
     def __contains__(self, x):
         x = simple_sympify(x)
+        if isinstance(x, SympyBoolean):
+            if x is sympy.true:
+                return self.upper == sympy.true
```

Comment:
`sympy.True in ValueRanges(-5, 5) -> False`. Could we change `if x is sympy.true` -> `if x is sympy.true and isinstance(self.lower, SympyBoolean) and isinstance(self.upper, SympyBoolean)`